src/KeyValueStore.js

```diff
@@ -11,6 +11,10 @@ class KeyValueStore extends Store {
     this._type = 'keyvalue'
   }
 
+  all () {
```

Comment:
Other stores use a get method to access `all` as a property. Should we also use `get` here?